### PR TITLE
Upgrade to replicated lib beta.13

### PR DESCRIPTION
**Pull request by jdewinne**
Changes:
* Upgrade replicated-lib to `0.0.1-beta.13`
* Added action create-object-store
* Added action create-postgres
* Added action expose-port

For review:
Anything under `dist` is generated and can be ignored.

### CODE REVIEW

File: create-object-store/action.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    required: true
+    default: ""
+  cluster-id:
+    description: "Cluster id to attach object store to"
+    required: true
+    default: ""
+  bucket-name:
+    description: "Name of the bucket to create"
+    required: true
+    default: ""
```
Comment (emosbaugh) [inline create-object-store/action.yml:15]: We should be consistent with the cli for all inputs

```suggestion
  bucket-prefix:
    description: "A prefix for the bucket name to be created"
    required: true
    default: ""
```

File: create-postgres/action.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    required: true
+    default: ""
+  version:
+    description: "Postgres version to provision."
+    required: false
+  instance-type:
+    description: "Instance type to provision"
+    required: false
+  disk:
+    description: "Disk size in GiB"
```
Comment (emosbaugh) [inline create-postgres/action.yml:19]: The cli sets the default to be 200 GiB. Do we need to do the same or will the api handle that?

**Review by emosbaugh — CHANGES_REQUESTED** [review]

File: create-object-store/action.yml (hunk 1 above)
Comment (jdewinne) [reply create-object-store/action.yml]: The cli uses `bucket` as flag

File: create-postgres/action.yml (hunk 2 above)
Comment (jdewinne) [reply create-postgres/action.yml:19]: Yes, if empty, the api should handle it.

**Review by emosbaugh — APPROVED** [review]

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**